python/test/gluon/test_lowerings.py

```diff
@@ -136,11 +134,9 @@ def test_reduce_layouts(M, N, src_layout, axis, epilogue_kind, dtype_str, saniti
     if src_layout == "linear_layout":
         src_layout = ttgl.DistributedLinearLayout(reg_bases=[[0, 16], [1, 0], [2, 0], [4, 0], [8, 0], [16, 0]],  #
                                                   lane_bases=[[0, 0], [0, 1], [0, 2], [0, 4], [0, 8]],  #
-                                                  warp_bases=[[32, 0], [0, 32]], block_bases=[], shape=[M, N])
```

Comment:
This is indeed correct. I'm curious why you applied different updates below and here...

Reply:
This will need to add an extra base when there are 64 threads as per https://github.com/triton-lang/triton/pull/7938#discussion_r2310330438. Once yo udo this you'll be able to remove the THREADS_PER_WARP skip below